.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,16 @@
+---
+name: '💡 Feature Request'
+about: Submit your idea for a change in the codebase.
+---
+
+# Feature Request
```

Comment:
@kelset What do you think after `Feature Request` we could add `Origin of the Request`? In that sense, we could understand the root cause of the request as well.

Reply:
Uhm I see what you are saying, maybe I can reword the "intro" section to include that.